tests/integration/test_api_gateway.py

```diff
@@ -84,6 +87,43 @@ def test_api_gateway_kinesis_integration(self):
         kinesis = aws_stack.connect_to_service('kinesis')
         kinesis.delete_stream(StreamName=self.TEST_STREAM_KINESIS_API_GW)
 
+    def test_api_gateway_sqs_integration_with_event_source(self):
+        # create target SQS stream
+        aws_stack.create_sqs_queue(self.TEST_SQS_QUEUE)
+
+        # create API Gateway and connect it to the target queue
+        result = connect_api_gateway_to_sqs(
+            'test_gateway4',
+            stage_name=self.TEST_STAGE_NAME,
+            queue_arn=self.TEST_SQS_QUEUE, path=self.API_PATH_DATA_INBOUND)
+
+        # create event source for sqs lambda processor
+        self.create_lambda_for_sqs_event_source(self.TEST_LAMBDA_HANDLER_NAME)
+        add_event_source(
+            self.TEST_LAMBDA_HANDLER_NAME,
+            'arn:aws:sqs:us-east-1:000000000000:test-sqs-queue-api-gw',
```

Comment:
nit: Can we replace the hardcoded ARN with `aws_stack.sqs_queue_arn(self.TEST_SQS_QUEUE)`

Reply:
ah nice  👍